WordPress/Classes/ViewRelated/Reader/ReaderDetailViewController.swift

```diff
@@ -765,22 +769,45 @@ open class ReaderDetailViewController: UIViewController, UIViewControllerRestora
         guard let post = post else {
             return
         }
+
         textView.isPrivate = post.isPrivate()
         textView.content = post.contentForDisplay()
 
-        // TODO: Get attributed string
-        // Modify the attributed string. Add top and bottom embeds
-        // Ensure formatting for embeds.
-        // Assign attributed string to textView.
-        // Besure that embed bounds are updated.
-        let attrStr = WPRichContentView.formattedAttributedStringForString(post.contentForDisplay())
-        let mAttrStr = NSMutableAttributedString(attributedString: attrStr)
+        updateRichText()
+        updateTextViewMargins()
+    }
+
+    private func updateRichText() {
+        guard let post = post else {
+            return
+        }
+
+        if #available(iOS 13, *) {
+            let isDark = traitCollection.userInterfaceStyle == .dark
+            textView.attributedText = isDark ? darkTextViewAttributedString : lightTextViewAttributedString
+        } else {
+            let attrStr = WPRichContentView.formattedAttributedStringForString(post.contentForDisplay())
+            textView.attributedText = attributedString(with: attrStr)
```

Comment:
It makes me a bit uncomfortable that we're pre-creating the attributed string on iOS 13 but generating it on demand on earlier versions. It would be interesting to pre-create it in both scenarios, with just the number of variants changing, but I can't think of a specific suggestion that doesn't seem to overcomplicate things ¯\\\_(ツ)\_/¯